README.md

```diff
@@ -840,7 +840,16 @@ In this example, `README.md` has conflicts. Open that file and look for the foll
 
 You will need to resolve the differences between the code that was added in your new commit (in the example, everything from the middle line to `new-commit`) and your `HEAD`.
 
-Sometimes these merges are complicated and you should use a visual diff editor:
+If you want to keep one branch's version of the code, you can use `--ours` or `--theirs`:
+
+```sh
+(master*)$ git checkout --ours README.md
+```
+
+- When *merging*, use `--ours` to keep changes from the current branch.
```

Comment:
Would it make more sense to say `..keep changes from the local current branch`?

Reply:
`local current branch` feels a little redundant, so I updated this line to use `local branch` 👍